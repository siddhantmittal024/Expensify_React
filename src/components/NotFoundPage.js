import { Switch, BrowserRouter, Route, Link, NavLink } from "react-router-dom";
import React from "react";

const NotFoundPage = () => (
  <div>
    404! - <Link to="/">Go Home</Link>
  </div>
);

export default NotFoundPage;
