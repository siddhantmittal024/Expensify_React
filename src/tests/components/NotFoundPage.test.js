import React from 'react';
import {shallow} from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should render ExpenseDashboardPage coorectly',()=>{
    const wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();
});