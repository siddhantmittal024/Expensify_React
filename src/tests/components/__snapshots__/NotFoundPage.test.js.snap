// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage coorectly 1`] = `
<div>
  404! - 
  <[object Object]
    to="/"
  >
    Go Home
  </[object Object]>
</div>
`;
